src/ansys/mapdl/core/cli/__init__.py

```diff
@@ -48,6 +49,21 @@ def main(ctx):
     main.add_command(stop)
     main.add_command(list_instances, name="list")
 
+    # HPC commands
+    # pymapdl hpc submit
+    # pymapdl hpc list
+    # pymapdl hpc stop
+    main.add_command(submit)
+
+    def old_pymapdl_convert_script_entry_point():
+        print(
+            """This CLI function has been deprecated. Use the following instead:
+
+pymapdl convert input_file.inp -o output_file.out ...
+
+For more information please visit: https://mapdl.docs.pyansys.com/version/dev/user_guide/cli.html
```

Comment:
```suggestion
For more information, see `PyMAPDL command line interface <https://mapdl.docs.pyansys.com/version/dev/user_guide/cli.html>`_.
```
Is there a reason, you don't use a :ref: role for a link?